### PR TITLE
Define more orchestrator build metadata

### PR DESCRIPTION
**Pull request by twaugh**
The koji-build-id label and repositories annotations are now defined.

Signed-off-by: Tim Waugh <twaugh@redhat.com>

### CODE REVIEW

**Comment by twaugh** [comment]
@csomh @lcarva

**Comment by twaugh** [comment]
I had to fix the indentation of that block which is why it looks like a larger change than it is.

File: docs/multiarch.rst [hunk 1]
```diff
@@ -711,56 +711,62 @@ This mechanism can also be used to temporarily disable a worker
...
-            ...
-          durations:
-            koji: ...
-            ...
-          errors: {}
-    plugins-metadata: '{"timestamps": {"orchestrate_build": "...", ...},
-      "durations": {"orchestrate_build": ..., ...}, "errors": {}}'
+    labels:
+      koji-build-id: ...
+    annotations:
```
Comment (lcarva) [inline docs/multiarch.rst]: Should we represent `repositories` here as well?

Comment (twaugh) [inline docs/multiarch.rst]: Updated to include that. While doing so, I noticed we also want `digests` (for sites without Koji integration).

**Comment by twaugh** [comment]
Updated to avoid duplicating information.

File: docs/multiarch.rst [hunk 2]
```diff
@@ -711,56 +711,82 @@ This mechanism can also be used to temporarily disable a worker
...
+              koji: ...
+              ...
+            durations:
+              koji: ...
+              ...
+            errors: {}
+      plugins-metadata: '{"timestamps": {"orchestrate_build": "...", ...},
+        "durations": {"orchestrate_build": ..., ...}, "errors": {}}'
+
+The existing koji-build-id label is an integer representing the
```
Comment (lcarva) [inline docs/multiarch.rst]: It looks like it's converted to a `str`.

Comment (twaugh) [inline docs/multiarch.rst]: Fixed.

**Pull request merged**